Dockerfile

```diff
@@ -75,8 +65,8 @@ COPY . /seed/
 COPY ./docker/wait-for-it.sh /usr/local/wait-for-it.sh
 
 # nginx configurations - alpine doesn't use the sites-available directory. Put seed
-# configuration file into the /etc/nginx/conf.d/ folder.
-COPY /docker/nginx-seed.conf /etc/nginx/conf.d/seed.conf
+# configuration file into the /etc/nginx/http.d/ folder.
+COPY /docker/nginx-seed.conf /etc/nginx/http.d/default.conf
```

Comment:
Apparently this is where our config file has to live now. conf.d is no longer by default loaded (and is loaded outside of the http directive so it fails there. http.d is now what's loaded in the default nginx config.